### PR TITLE
Assignments and categories

### PR DESCRIPTION
**Pull request by MeesvanStraten**
Fixed some problems

### CODE REVIEW

**Comment by waltersajtos** [comment]
Looks good to me but my code is in here as well (:

File: Api/Controllers/CategoryController.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿using Api.InputOutputModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Models;
+using Services;
+using System.Threading.Tasks;
+
+namespace Api.Controllers
+{
+    [Route("[controller]")]
```
Comment (Ruby77) [inline Api/Controllers/CategoryController.cs:10]: Same here, not the same attribute

File: Api/Controllers/CategoryController.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+            categoryService.Update(newCategory);
+            categoryService.Save();
+
+            return Ok(newCategory);
+        }
+
+        // DELETE api/<ValuesController>/5
+        [HttpDelete("{id}")]
+        public void Delete(int id)
```
Comment (Ruby77) [inline Api/Controllers/CategoryController.cs:65]: Why is this a void? Shouldn't we  return a HTTP status?

File: Data/Migrations/20210107130909_fixedAssignmentTypes.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        {
+
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
+        }
+    }
+}
```
Comment (Ruby77) [inline Data/Migrations/20210107130909_fixedAssignmentTypes.cs:17]: Empty migration?

**Review by Ruby77 — CHANGES_REQUESTED** [review]
Looks good guys!

File: Api/Controllers/CategoryController.cs (hunk 2 above)
Comment (waltersajtos) [reply Api/Controllers/CategoryController.cs:65]: We didn't make the value controller, it think it came from your branch, no?

**Review by Ruby77 — DISMISSED** [review]

**Review by waltersajtos — APPROVED** [review]

**Pull request merged**